papaparse.js

```diff
@@ -1165,20 +1170,19 @@ License: MIT
 			return value;
 		}
 
-		function applyHeaderAndDynamicTypingAndTransformation()
+		function applyHeaderAndDynamicTypingAndTransformation(isStep)
 		{
 			if (!_results || (!_config.header && !_config.dynamicTyping && !_config.transform))
 				return _results;
 
-			for (var i = 0; i < _results.data.length; i++)
+			function processRow(rowSource, i)
```

Comment:
Could you avoid refactoring the code into function instead of fors?

I do not say i don't like it but i like to keep commits simple and related to the issue. 

Please open a new PR for this kind of refactoring.

Reply:
I was not intending to refactor for refactors sake. If the loop body is not put into a function, it will need to be repeated. I attempted to make the change as minimal as possible.